4_Dependency.md

```diff
@@ -1,10 +1,15 @@
 # Metric: Dependency
 
-Activity Metric | Description
---- | ---
- ... | ...
- ... | ...
- ... | ...
+ #Activity Metric | Description
+---|---
+Number of External Components Relied Upon| 1st of several ways to measure upstream reliance
+Depth of Reliance on External Components| ? informed by activity metrics or defined by the number "volume" of connections, calls, etc. between the same components

```

Comment:
"depth" in networks means levels of calls. The description is more about how much external components are used.

Maybe we can call this "Use of external components" with the description "Number of connections, calls, references, or other uses of external components. Informs how much a project relies on external components."